templates/osbs-buildroot-imagestream.yml.j2

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  annotations:
+    openshift.io/image.insecureRepository: "true"
+  name: osbs-buildroot
+  namespace: {{ osbs_namespace }}
+spec:
+  dockerImageRepository: brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rcm/buildroot
```

Comment:
Exposing internal service address is a bad idea.

Also, there is no particular tag to follow

Reply:
Yeah, this should be a variable. But we don't need to add a tag here. We actually want to tag the whole repo.

Reply:
Every tag in that repo would be imported and a new ImageStreamTag would be created - it would affect the performance.

A better way would be creating a blank ImageStream and tagging latest to a particular buildroot image in the next step. This way we'll have a proper history of buildroots used without extra tags imported

Reply:
There's a maximum, by default I believe is 5.

Reply:
Ah, correct, its determined by `MaxImagesBulkImportedPerRepository`, which is 5 by default, so its not that important.

Reply:
ok changed it to variable